src/Module.jl

```diff
@@ -26,6 +26,27 @@ function check_parent(M::FPModule{T}, N::FPModule{T}) where T <: RingElement
 end
 
 is_finite(M::FPModule{<:FinFieldElem}) = true
+is_finitely_generated(M::FPModule) = true
+is_finitely_generated(M::Module) = isfinite(ngens(M)) || throw(NotImplementedError(:is_finitely_generated, M))
+
+@doc raw"""
+    is_noetherian(M::Module)
+
+Check if the module $M$ is Noetherian. This is currently implemented only for finitely generated modules
+in which case it is sufficient to check whether the base ring is Noetherian.
```

Comment:
```suggestion
Check if the module $M$ is Noetherian.
```
I don't think we should include this here: if e.g. OSCAR adds a method dealing with some other kinds of modules then this comment becomes wrong. It also is made wrong by the change I suggest below.

We could include a warning like 'This may not be implemented for all modules.' but then we don't do that for many other functions where this also holds (just for rings/groups/... instead of modules) ...